examples/conditionformat2/manifest.json

```
@@ -14,9 +14,9 @@
     },
     "icon": "image/icon.png",
     "homepage_url": {
-        "ja": "https://developer.cybozu.io/hc/ja/articles/208236353",
-        "en": "https://developer.kintone.io/hc/en-us/articles/115009635868",
-        "zh": "https://cybozudev.kf5.com/hc/kb/article/1011206/"
+        "ja": "https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/conditionformat-v2-plugin/",
+        "en": "https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/conditionformat-v2-plugin/",
+        "zh": "https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/conditionformat-v2-plugin/"
```

```suggestion
        "ja": "https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/conditionformat-v2-plugin/",
        "en": "https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/conditionformat-v2-plugin/",
        "zh": "https://cybozudev.kf5.com/hc/kb/article/1011206/"
```

簡体字は別のページは存在するのでそのままにしてもらえると嬉しいです！